README.md

```diff
@@ -383,11 +383,9 @@ You can use the scoped key created in Ruby for API requests from any client. Sco
 
 #### Access Keys
 
-You can use access keys to restrict the functionality of a key you use with the Keen API. Access keys can also enrich events that you send.
+You can use [access keys](https://keen.io/docs/api/?ruby#access-keys) to restrict the functionality of a key you use with the Keen API. Access keys can also enrich events that you send.
```

Comment:
This is probably going to sound nitpicky, but we try to consistently capitalize "Access Keys."

And you couldn't possibly know that since we have no docs style guide. :) I'm happy to make the fixes if you want.

Reply:
I'm happy to take care of this myself, though I don't mind if you want to. I'm still spun up on this project so should be no big deal, though I won't be back to open source time until this evening or tomorrow. If this PR is still open when I shift back into open source time, I'll jump right on it.

Reply:
Sounds good. I check back in to merge tomorrow.

Reply:
Ok this PR is updated with enforced capitalization on all instances of "Access Keys" I found.